.gitignore

```diff
@@ -3,6 +3,9 @@
 
 # Editors
 *~
+*.#
+*/**/#*
+*/**/.#*
```

Comment:
Could you undo changes here? They seem too broad.
You can use local excludes for this kind of editor-specific ignores: https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally